### PR TITLE
can now only conquer orthogonally adjacent squares.

### PR DESCRIPTION
**Pull request by gaguevaras**

### CODE REVIEW

**Comment by gaguevaras** [comment]
@slashman yo. sup. can now only conquer orthogonally adjacent squares. yunomerge?

File: client/index.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
 <script>
   var waiting = false;
   var color = 'black';
-  var socket = io('http://games.slashware.net:3002');
+  var socket = io('http://localhost:3001');
```
Comment (slashman) [inline client/index.html:16]: Note to self: We should move this to a cfg file

File: server/index.js [hunk 2]
```diff
@@ -32,26 +32,49 @@ io.on('connection', function(socket){
 		var lastPlayerAction = lastActions[socket.id];
 		if (lastPlayerAction && new Date().getTime() - lastPlayerAction < 1000){
 			return;
+		}		
+		console.log('can conquer', canConquer(lastPlayerAction, model.map, where, colors[socket.id]));
```
Comment (slashman) [inline server/index.js]: Let's remove this debug message

File: server/index.js [hunk 3]
```diff
@@ -32,26 +32,49 @@ io.on('connection', function(socket){
...
+		if(canConquer(lastPlayerAction, model.map, where, colors[socket.id])){
+			socket.broadcast.emit('conquered', {
```
Comment (slashman) [inline server/index.js:37]: Note to self: Change this to use a single emit and reach all.

File: server/index.js [hunk 4]
```diff
@@ -32,26 +32,49 @@ io.on('connection', function(socket){
...
-			y: where.y,
-			color: colors[socket.id]
-		});
-		model.map[where.x][where.y] = colors[socket.id];
 	});
 
 });
 
+var canConquer = function (lastAction, map, where, color){
+	if(!lastAction){
```
Comment (slashman) [inline server/index.js:58]: This check is not needed, I think the lastAction param is not needed here either

Comment (gaguevaras) [inline server/index.js:58]: it's needed for the first region to be conquered

**Comment by gaguevaras** [comment]
@slashman done!

**Pull request merged**